builds/misc/images-release.yaml

```diff
@@ -29,7 +29,6 @@ jobs:
           args: "-c Release"
           scriptPath: scripts/linux/buildBranch.sh
       - template: templates/build-broker-watchdog.yaml
-      - template: templates/consolidate-build-artifacts.yaml
```

Comment:
I removed this file and moved the relevant logic for the edgehub build context setup into `templates/build-broker-watchdog.yaml`